Misc/NEWS.d/next/Library/2023-04-14-21-12-32.gh-issue-103538.M4FK_v.rst

```diff
@@ -0,0 +1,4 @@
+Remove code defined by the ``TK_AQUA`` macro, because:
+
+#. it reportedly was only needed for Tk 8.4.7 or earlier (see gh-issue-49370), whereas Tkinter has since required 8.5.12 (see gh-issue-91152); and
+#. it seems to have been unused since it was committed (in gh-issue-35685), as neither CPython nor Tcl/Tk ever defined the ``TK_AQUA`` macro anywhere.

```

Comment:
The bulleted list would probably be fine, but we can condense and add a bit of extra info anyway:

```suggestion
Remove ``_tkinter`` module code guarded by definition of the ``TK_AQUA`` macro
which was only needed for Tk 8.4.7 or earlier and was never actually defined by
any build system or documented for manual use.
```